syntax ="proto3";
package IM;
message SignUp
{
      int32 networkType=1;
      int32 msgID=2;//每条消息的标识
      int32 recvID=3;
      int32 sendID=4;
      bytes password=5;
      bytes nickname=6; //昵称
      bytes phone=7;
	  int32 sex=8;
      bytes icon=9;   //头像
	  int32 state=10;  //1:注册成功 2：该号码已注册 3：
}